content/ecosystem/_index.html

```diff
@@ -0,0 +1,461 @@
+
+        </div>
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-terminal"></i> cuCIM </h2>
+          <div class="tags-container border-top has-border-white">
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> COMPUTER VISION </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> IMAGE PROCESSING </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PYTHON </span>
+          </div>
+          <p class="has-text-white"> Greatly accelerate your computer vision and image processing tasks, especially for biomedical imaging.</p>
+          <a href="https://github.com/rapidsai/cucim">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/cucim/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i>
+          </a> <br>
+          <a href="https://github.com/orgs/rapidsai/projects/62/views/1">Roadmap <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+        </div>
+      </div>
+
+      <div class="columns mt-5">
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-terminal"></i> Many More...</h2>
+          <p class="has-text-white"> Visit our <a href="https://github.com/rapidsai" target="_blank"> GitHub organization page</a> or our <a href="https://docs.rapids.ai/api" taget="_blank">documentation </a> for a more extensive list of projects.</p>
+        </div>
+        <!-- spacer -->
+        <div class="column"></div>
+      </div>
+
+    </div>
+  </section>
+
+  <div id="rapids-integration"></div>
+  <div class="bottom-slant-container">
+    <div class="bottom-slant-up has-background-rapids-purple-primary"></div>
+  </div>
+
+  <!-- RAPIDS Integrated -->
+  <section class="section has-background-rapids-gray padding-top-slant padding-bottom-slant">
+    <div class="container">
+      <h1>RAPIDS Integration</h1>
+      <p class="subheading mb-5">Projects and services that have RAPIDS acceleration as an added feature. </p>
+
+      <div class="columns">
+        <div class="column is-one-half">
+          <img class="image img-w100" src="/images/spark-logo-trademark.png">
+          <h2><i class="fa-regular fa-code-merge"></i> Spark RAPIDS</h2>
+          <div class="tags-container border-top has-border-rapids-purple-primary ">
+            <span class="tags-item has-text-rapids-purple-primary  has-border-rapids-purple-primary "><i class="fa-solid fa-tag"></i> APACHE SPARK </span>
+            <span class="tags-item has-text-rapids-purple-primary  has-border-rapids-purple-primary "><i class="fa-solid fa-tag"></i> SQL</span>
+          </div>
+          <p> NVIDIA is bringing RAPIDS to Apache Spark to accelerate ETL workflows with GPUs.</p>
```

Comment:
I think we should mention the benefits here--speed, cost, drop-in replacement

```suggestion
          <p> NVIDIA is bringing RAPIDS to Apache Spark to GPU-accelerate ETL workflows with no code change, improving performance and reducing total costs.</p>
```

Reply:
Good call @aravenel. One more tweak: "NVIDIA is bringing" sounds like a future plan. Can we emphasize more strongly that it's a ready-to-use product?

```suggestion
          <p> NVIDIA integrates RAPIDS with Apache Spark to GPU-accelerate ETL workflows with no code change, improved performance, and reduced total cost.</p>
```